.github/workflows/ci.yml

```diff
@@ -85,21 +87,24 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
 
-      - name: Install scopelint
-        uses: engineerd/configurator@v0.0.8
-        with:
-          name: scopelint
-          repo: ScopeLift/scopelint
-          fromGitHubReleases: true
-          version: latest
-          pathInArchive: scopelint-x86_64-linux/scopelint
-          urlTemplate: https://github.com/ScopeLift/scopelint/releases/download/{{version}}/scopelint-x86_64-linux.tar.xz
-          token: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Check formatting
-        run: |
-          scopelint --version
-          scopelint check
+        run: forge fmt --check
+
+      #- name: Install scopelint
```

Comment:
Why aren't we using scopelint in this repo? If we're permanently not using it we can delete this instead of commenting it out. If we're planning to bring it back let's have an issue to track

Reply:
Mainly because we can't ignore linting on certain lines. Once this is implemented in scopelint we can comment this back in.